packages/cra-template-edu/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@chanzuckerberg/cra-template-edu",
+  "version": "0.1.0",
+  "license": "MIT",
+  "keywords": [
+    "react",
+    "create-react-app"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/chanzuckerberg/frontend-libs"
+  },
+  "homepage": "https://github.com/chanzuckerberg/frontend-libs/blob/main/packages/cra-template-edu",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "template",
+    "template.json"
+  ],
+  "devDependencies": {
+    "@chanzuckerberg/eds": "^5.3.1",
+    "@chanzuckerberg/prettier-config-edu": "workspace:*",
+    "@chanzuckerberg/story-utils": "workspace:*",
+    "@storybook/testing-react": "^1.3.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/react-dom": "^18.0.8",
+    "clsx": "^1.2.1",
+    "react-router-dom": "^6.4.2"
```

Comment:
This is the package.json of the cra-template-edu package, NOT the generated app... gets kind of confusing.

Unfortunately had to list the generated app's dependencies here, since they're referenced in the template files. Good thing is TypeScript and ESLint will still analyze the template files.